wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceValidator.java

```diff
@@ -48,6 +52,9 @@
   private static final Pattern VOLUME_NAME = Pattern.compile("[a-z][a-z0-9]{1,18}");
   private static final Pattern VOLUME_PATH = Pattern.compile("/.+");
 
+  static final String PLUGINS_ATTRIBUTE = "plugins";
```

Comment:
It might be better to use existing constants https://github.com/eclipse/che/blob/174e1aea9b5fc68815da7e7a485b6ff51b38ba15/wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java#L91

Reply:
Ah didn't see, thanks. Will fix.